content/en/about/day-in-life-dev.md

```diff
@@ -6,95 +6,111 @@ position: 7
 category: 'About Us'
 ---
 
-Potential candidates often wonder what a typical day looks like at Narrative, and this page is an attempt to answer the question.
+Potential candidates often wonder what a typical day looks like at Narrative, and this page is an attempt to answer the
+question.
 
 ## Sync with the rest of the team
+
 ### \#daily-gsd
 
-We use [Slack](https://slack.com/) to stay connected with the rest of the team. 
+We use [Slack](https://slack.com/) to stay connected with the rest of the team.
 
-The `#daily-gsd` channel lets us keep track of what our teammates are working on and identify blockers as well as potential conflicts.
-Each one of us starts the day by taking a look at his/her work queue, what (s)he achieved yesterday, and what (s)he plans to complete today, 
-and writes a 2-3-liner in `#daily-gsd` to let the rest of the team know about the current priorities.
+The `#daily-gsd` channel lets us keep track of what our teammates are working on and identify blockers as well as
+potential conflicts.
+Each of us starts the day by taking a look at our work queue, what we achieved yesterday, and what we plan to complete
+today, then we write a couple sentences in `#daily-gsd` to let the rest of the team know about the current priorities.
 
 ### \#auto-techops
 
-The person who is on techops rotation will look at the various warnings and alerts in the `#auto-techops` channel in order to identify 
-things that need to be fixed, failed clusters, etc.
-
-### \#partner-success and \#helpdesk
+The person who is on call will look at the various warnings and alerts in the `#auto-techops` channel in order to
+identify things that need to be fixed, failed clusters, etc.
 
-The person who is on Partner Success (PS) rotation will monitor the `#partner-success` and `#helpdesk` channels, as well as the Partner 
-Success queue in [Clubhouse](http://clubhouse.io/) in order to determine the next priorities.
+The on call will also monitor the `#partner-success` and `#helpdesk` channels to prevent partners and the business from
+beingn blocked on technical issues.
```

Comment:
being

Reply:
Thanks Sami. Fixed in [199a36c](https://github.com/narrative-io/narrative-jobs/pull/4/commits/199a36c6a0d366b9c3b4c47f6d7b28018978c1a5)